test/tests.js

```diff
@@ -1,11 +1,16 @@
+const NODE_VERSION = process.version.split('.');
+const NODE_MAJOR_VERSION = parseInt(NODE_VERSION[0].replace(/\D/g, ''));
+
 process.env.NODE_ENV = 'test';
 
-require('../packages/graphql-server-core/dist/runQuery.test.js');
-require('../packages/graphql-server-module-operation-store/dist/operationStore.test');
-require('../packages/graphql-server-express/dist/expressApollo.test');
-require('../packages/graphql-server-express/dist/connectApollo.test');
-require('../packages/graphql-server-hapi/dist/hapiApollo.test');
-require('../packages/graphql-server-koa/dist/koaApollo.test');
-require('../packages/graphql-server-restify/dist/restifyApollo.test');
-require('../packages/graphql-server-lambda/dist/lambdaApollo.test');
-require('../packages/graphql-server-express/dist/apolloServerHttp.test');
+require('../packages/graphql-server-core/src/runQuery.test');
+require('../packages/graphql-server-module-operation-store/src/operationStore.test');
+require('../packages/graphql-server-express/src/expressApollo.test');
+require('../packages/graphql-server-express/src/connectApollo.test');
+require('../packages/graphql-server-hapi/src/hapiApollo.test');
+if (NODE_MAJOR_VERSION >= 6) {
```

Comment:
Please open an issue about that, and then add a comment pointing to that issue, it should be fixed as well.